### ISSUE TITLE
Do we really need the derivative of the activation function for gradient descent?

### ISSUE THREAD

**Issue by ssocolow** [ssocolow/Neural-Network-Saves#2]
In the Coding Train's series on [Neural Networks](https://www.youtube.com/watch?v=8H2ODPNxEgA&list=PLRqwX-V7Uu6aCibgK1PTWWu9by6XFdCfh&index=16), he uses the derivative of the sigmoid function but I am not sure if we really need to use it because the errors can tell us which way the weights need to be adjusted.

This may just be because I haven't taken calculus and don't really understand the derivative.
